apps/src/templates/studentSnapshot/StudentSnapshot.tsx

```diff
@@ -90,21 +90,49 @@ const StudentSnapshot: React.FC = () => {
         </div>
       </div>
 
-      <Typography variant="h2" className={styles.studentNameHeader}>
+      <Typography
+        variant="h4"
```

Comment:
Now that I see this comment, I'm curious about the semantic structure of these pages.
I am assuming an h4 is more semantically meaningful here, but I lack context for what's on the rest of the page.

Reply:
Mostly just following Molly's for the page on this. I know the h1 is the `student snapshot` name. and most other content on the page will be body text or labels. I don't think we will run into any issues